### ISSUE TITLE
Non existent formkit, because cannot resolve and then when you put it into the component it fails

### ISSUE THREAD

**Issue by sibalonat** [GustavoFenilli/formkit-addon-inertia#11]
I wanted to say that from what I learned this seems a very good work. Although I cannot provide a proper starting configuration to make the plugin work. For some reason, it fails to resolve and then it doesn't setup the component.

**Comment by sibalonat** [comment]
yeah never mind. For else is facing this. In inertia there is a plugin that is passed so adding a plugin and defaultConfig doesn't work. But any other name does. 
the example I found from StackOverflow
import { plugin as FKp, defaultConfig as FKdc } from "@formkit/vue";

**Comment by GustavoFenilli** [comment]
So you made it work? can you share what the code for your config looks like so I can see if there is something that could be improved?

**Comment by sibalonat** [comment]
Yes.
on the main.js on a Laravel application.
// formkit
instead of 
`import { plugin, defaultConfig } from "@formkit/vue";`
you do 
`import { plugin as FKp, defaultConfig as FKdc } from "@formkit/vue";`

This is because of inertia when creates and setups the app, with
```
createInertiaApp({
   setup({ el, App, props, plugin }) {
    
   }
}) 
```
the plugin there could not be used as the package's name, therefore you need an alias for that. 

For some reason this was happening also locally, don't know why. But if you register this globally it would be fine
